src/scripts/drag-text.js

```diff
@@ -85,6 +85,7 @@ H5P.DragText = (function ($, Question, ConfirmationDialog) {
       submitAnswer: "Submit",
       tryAgain: "Retry",
       behaviour: {
+	numLines: null,
```

Comment:
1. Please mind the formatting. The indentation is off here.
2. `numLines` is supposed to be a number, so sanitizing it to `null` doesn't make sense. You don't have to set it here at all, however, as the field says it's optional.

Reply:
Fair point